scst-store/amr-graphql-query.hbs.md

```diff
@@ -0,0 +1,255 @@
+# AMR Graphql Querying
+
+This page contains information on how to connect to the graphql playground, and how to query each supported data model.
+See [AMR Data Models](amr-data-model.hbs.md) for more information on the AMR data models.
```

Comment:
I'm not seeing this file in docs-tap/main or in this fork. Where is it?

Reply:
This was in the other data model branch that I made, I wanted to just keep this MR focused on this file specifically. But actually now I feel like it might be a bit confusing, I will merge the two MRs together along with rebasing with the new structural amr changes